egs/wsj/s5/steps/tfrnnlm/lmrescore_rnnlm_lat_pruned.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# Copyright 2015  Guoguo Chen
+#           2017  Hainan Xu
+# Apache 2.0
+
+# This script rescores lattices with RNNLM trained with TensorFlow.
+# It uses a pruend algorithm to speed up the runtime and improve the accuracy.
+# which is an improved version over steps/tfrnnlm/lmrescore_rnnlm_lat.sh,
+# which uses the exact same interface
+# The details of the pruning algorithm is described in
+# http://www.cs.jhu.edu/~hxu/tf.pdf
```

Comment:
this may not be a very permanent location.  My homepage is more permanent, it better to put it there, you can send me the pdf if it's not there already(remind me where it's being published).